### ISSUE TITLE
Pagination system

### ISSUE THREAD

**Issue by jvm-odoo** [saadpasta/react-blog-github#28]
If a blog has a lot of articles, we need to set up some pagination to avoid infinite scrolling

**Comment by saadpasta** [comment]
Good Idea! We could intersection observer for that or something else we can add this to the Todo Section

**Comment by Freedooom111** [comment]
H de HD z to HD s this

**Comment by Deepak-Kumar9891** [comment]
good idea...
